<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1412</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>1331</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select an image for encryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_crypto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_base_image">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>1151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selected picture:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose_image_decrypto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select a picture to decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decrypto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_decrypto_image">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>1151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selected picture for decryption:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose_keys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select decryption keys</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_keys">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>680</y>
      <width>1151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected key file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export decrypted text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_crypto">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>600</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_decrypto">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>720</y>
      <width>1271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1412</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
